packages/form/src/Datepicker/Datepicker.js

```diff
@@ -64,7 +64,7 @@ class Datepicker extends Component<Props> {
 		const {open} = this.props;
 
 		Moment.updateLocale('nl', {
-			months : ["januari", "februari", "maart", "april", "mei", "juni", "juli", "augustus", "september", "oktober", "november", "december"],
+			months : ["Januari", "Februari", "Maart", "April", "Mei", "Juni", "Juli", "Augustus", "September", "Oktober", "November", "December"],
```

Comment:
Maanden en dagen in het Nederlands zijn niet met een hoofdletter.

Reply:
Fixed with CSS ;-)